src/bin/fullnode.rs

```diff
@@ -52,7 +52,13 @@ fn main() {
                 .value_name("DIR")
                 .takes_value(true)
                 .required(true)
-                .help("use DIR as persistent ledger location"),
+                .help("Use DIR as persistent ledger location"),
+        ).arg(
+            Arg::with_name("rpc")
```

Comment:
I don't see anything using the `--rpc` argument (perhaps around https://github.com/solana-labs/solana/blob/master/multinode-demo/leader.sh#L36).  Does this mean that by default there will be no RPC port enabled?

Reply:
If `--rpc` is not set, the node will use the default RPC port (as determined in `Node::new_with_external_ip`). At present, there will be an RPC port enabled on every node boot; either custom port from this argument, default RPC port, or random available port.

Reply:
cool, thanks